components/org.wso2.carbon.identity.api.server.dcr/src/main/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/util/DCRMUtils.java

```diff
@@ -151,6 +151,9 @@ public static ApplicationDTO getApplicationDTOFromApplication(Application applic
         applicationDTO.setClientSecret(application.getClientSecret());
         applicationDTO.setRedirectUris(application.getRedirectUris());
         applicationDTO.setGrantTypes(application.getGrantTypes());
+        applicationDTO.setClientSecretExpiresAt(0L); // currently, we are not setting an expiration time for
+        // the client secret, hence according to the DCR specification we have to set the expiration time to 0.
```

Comment:
Let's add the comment on top of `applicationDTO.setClientSecretExpiresAt`. Comment should start with caps

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1847